assets/js/components/data-table.js

```diff
@@ -75,7 +72,11 @@ export const getDataTableFromData = ( data, headers, options ) => {
 						? <div className="googlesitekit-table__body-item-content">
 							<Link
 								className="googlesitekit-table__body-item-link"
-								href={ useAdminURLs ? getSiteKitAdminURL( 'googlesitekit-dashboard', { permaLink } ) : permaLink }
+								href={
+									useAdminURLs
+										? getSiteKitAdminURL( 'googlesitekit-dashboard', { permaLink: global._googlesitekitLegacyData.admin.siteURL + link } )
```

Comment:
This works in the way we currently use this function, but it could still cause issues. `link` is not necessarily a relative URL - for example on the Analytics module page, it is a URL to the Analytics FE.

To be safe here, we should **only** prefix `link` with the site URL if it is a relative URL (starting in `/`).

Reply:
Ok, updated.